From 3eea44d9f627e670160aaa894845da9eb4d16327 Mon Sep 17 00:00:00 2001
From: Steve Cornelius <steve.cornelius@freescale.com>
Date: Mon, 18 Jun 2012 15:43:11 -0700
Subject: [PATCH 1657/2463] ENGR00215228-1: Remove unused DECO base pointer

Remove unused DECO base pointer to be consistent with staged-for-1.2
QorIQ release.

Signed-off-by: Steve Cornelius <steve.cornelius@freescale.com>
---
 drivers/crypto/caam/ctrl.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/drivers/crypto/caam/ctrl.c b/drivers/crypto/caam/ctrl.c
index 7c11320..3a245aa 100644
--- a/drivers/crypto/caam/ctrl.c
+++ b/drivers/crypto/caam/ctrl.c
@@ -58,7 +58,6 @@ static int caam_probe(struct platform_device *pdev)
 	struct caam_ctrl __iomem *ctrl;
 	struct caam_full __iomem *topregs;
 	struct caam_drv_private *ctrlpriv;
-	struct caam_deco **deco;
 	u32 deconum;
 #ifdef CONFIG_DEBUG_FS
 	struct caam_perfmon *perfmon;
@@ -166,9 +165,6 @@ static int caam_probe(struct platform_device *pdev)
 	ctrlpriv->deco = kmalloc(deconum * sizeof(struct caam_deco *),
 				 GFP_KERNEL);
 
-	deco = (struct caam_deco __force **)&topregs->deco;
-	for (d = 0; d < deconum; d++)
-		ctrlpriv->deco[d] = deco[d];
 	/*
 	 * Detect and enable JobRs
 	 * First, find out how many ring spec'ed, allocate references
-- 
1.7.7.4

